DuckDuckGo/SourcesForSwiftPackage/Logic/PrivacyDashboardLogic.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  PrivacyDashboardLogic.swift
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import WebKit
+import Combine
+
+public final class PrivacyDashboardLogic: NSObject {
```

Comment:
Not keen on this name tbh.  Is there something better, something that describes the responsibility of this class?  Feels more like a mediator?